framework/src/modules/chain/blocks/transactions/votes_weight.js

```diff
@@ -37,14 +37,21 @@ const updateDelegateVote = (
 	const delegateAccount = stateStore.account.get(delegateAddress);
 	const voteBigNum = new BigNum(delegateAccount.voteWeight || '0');
 	const voteWeight = voteBigNum[method](amount).toString();
-	const updatedDelegateAccount = {
-		...delegateAccount,
-		voteWeight,
-	};
-	stateStore.account.set(delegateAddress, updatedDelegateAccount);
+	delegateAccount.voteWeight = voteWeight;
```

Comment:
I think this change didn't actually affect so much

Reply:
Yeah it's just better to read IMO